fili-core/src/test/resources/testApplicationConfig.properties

```diff
@@ -66,3 +66,6 @@ bard__intersection_reporting_enabled = false
 
 # Flag to turn on case sensitive keys in keyvalue store
 bard__case_sensitive_keys_enabled = false
+
+# Default value for Druid limit on uncovered interval
+bard__druid_uncovered_interval_limit = -1
```

Comment:
It is not needed, but is it good to have? So that this config is easily found and change instead of having to go to the code and look for the config parameter and default.